float32[] v_traj
float32[] w_traj
float32[] x_traj      # computed x-position trajectory
float32[] y_traj      # computed y-position trajectory
